modules/serverless-rn-1-33-0.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-33-0_{context}"]
+= Red Hat {ServerlessProductName} 1.33
+
+{ServerlessProductName} 1.33 is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes.
+
+[id="new-features-1-33-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.12.
+* {ServerlessProductName} now uses Knative Eventing 1.12.
+* {ServerlessProductName} now uses Kourier 1.12.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.12.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.12.
+* The `kn func` CLI plugin now uses `func` 1.13.
+
+* OpenShift Serverless on ARM64 is now available as Technology Preview (TP).
```

Comment:
Just "Technology Preview", per https://redhat-documentation.github.io/supplementary-style-guide/#technology-preview:
```suggestion
* OpenShift Serverless on ARM64 is now available as Technology Preview.
```
or, probably more correct:
```suggestion
* OpenShift Serverless on ARM64 is now available as a Technology Preview feature.
```

Reply:
The upper option ("as Technology Preview") is better, because it's somewhat of a misnomer to call support for an arch a feature.